engine/pom.xml

```diff
@@ -23,7 +23,7 @@
     <distributionManagement>
         <site>
             <id>site</id>
-            <url>http://validator.hibernate.org</url>
+            <url>http://hibernate.org/validator</url>
```

Comment:
> There is no ORM, because ORM == hibernate.org (of course one could argue that such a redirect would make sense)

No that's a problem we need to fight.
We should be working hard on making sure that Hibernate.org == Java data platform.
When I asked why we don't have an orm.hibernate.org @emmanuelbernard explained me that it's because the subdomains are legacy.. makes sense to me, but it suggests you should try stopping using them.